packages/material-ui-lab/src/utils/helpers.js

```diff
@@ -0,0 +1,47 @@
+import warning from 'warning';
+
+// It should to be noted that this function isn't equivalent to `text-transform: capitalize`.
+//
+// A strict capitalization should uppercase the first letter of each word a the sentence.
+// We only handle the first word.
+export function capitalize(string) {
+  if (process.env.NODE_ENV !== 'production' && typeof string !== 'string') {
+    throw new Error('Material-UI: capitalize(string) expects a string argument.');
+  }
+
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+export function contains(obj, pred) {
+  return Object.keys(pred).every(key => obj.hasOwnProperty(key) && obj[key] === pred[key]);
+}
+
+/**
+ * Safe chained function
+ *
+ * Will only create a new function if needed,
+ * otherwise will pass back existing functions or null.
+ *
+ * @param {function} functions to chain
+ * @returns {function|null}
+ */
+export function createChainedFunction(...funcs) {
```

Comment:
`createChainedFunction` isn't used in ToggleButton, it can be deleted.

And if `capitalize` is to be used by multiple packages, it should probably go in the utils package.  But do we really need to reinvent the wheel here? Surely capitalize is already a thing? CC: @oliviertassinari